mmdet3d/datasets/pipelines/loading.py

```diff
@@ -123,25 +151,35 @@ def __call__(self, results):
                 - points (np.ndarray): Multi-sweep point cloud arrays.
         """
         points = results['points']
-        points[:, 3] /= 255
         points[:, 4] = 0
         sweep_points_list = [points]
         ts = results['timestamp']
-
-        for idx, sweep in enumerate(results['sweeps']):
-            if idx >= self.sweeps_num:
-                break
-            points_sweep = self._load_points(sweep['data_path'])
-            points_sweep = np.copy(points_sweep).reshape(-1, self.load_dim)
-            sweep_ts = sweep['timestamp'] / 1e6
-            points_sweep[:, 3] /= 255
-            points_sweep[:, :3] = points_sweep[:, :3] @ sweep[
-                'sensor2lidar_rotation'].T
-            points_sweep[:, :3] += sweep['sensor2lidar_translation']
-            points_sweep[:, 4] = ts - sweep_ts
-            sweep_points_list.append(points_sweep)
-
-        points = np.concatenate(sweep_points_list, axis=0)[:, [0, 1, 2, 4]]
+        if self.pad_empty_sweeps and len(results['sweeps']) == 0:
+            for i in range(self.sweeps_num):
+                if self.remove_close:
+                    sweep_points_list.append(self._remove_close(points))
+                else:
+                    sweep_points_list.append(points)
+        else:
+            if len(results['sweeps']) <= self.sweeps_num:
+                choices = [i for i in range(len(results['sweeps']))]
```

Comment:
`[i for i in range(len(results['sweeps']))]` -> `np.range(len(results['sweeps']))`